### DISCUSSION TITLE
3 Months Subscription

### DISCUSSION THREAD

**Discussion by surtic86** [Glench/ExtPay#58] [Ideas]
Well i would like to have the Option for a 3 Months Subscription. Is that already Possible on some way?

Benefits for the Users: 

- I can give them a Reduction of the Price
- More options for them (i also Plan to give then a Year Subscription)

Benefits for me: 

- Less money goes to Stripe just one Transaction instead of 3
- Less Problem with "no Found" on Credit Card on the Customers

**Comment by connallykendrick** [comment]
I would potentially like this option as well if possible. 

Side question @Glench  since we're in this thread - 

If price were to change from say, $10/mth to $15/mth - Are ALL users affected by this or only new users?
Visa versa, $15/mth to $10/mth

**Reply by connallykendrick** [reply]
I just answered my own question here - 
![price](https://user-images.githubusercontent.com/86803661/194932887-4d50a5d2-ff8e-43fd-8042-9bc69019c13f.PNG)

However, is there any correct way of going about a price increase for ALL users?

**Comment by bergwerf** [comment]
I would find a 3-month subscription option interesting as well.

**Comment by rurimiao** [comment]
3 Months Subscription as well as 6 Months Subscription is needed! stripe fees(2.9% + 30 cents) is just too heavy!

**Reply by rafalkrupinski** [reply]
The commission difference between  for 1 and 3 months is 60 cent. What market are you on that you can't offload it to the users? ;)
